x-pack/plugins/lens/public/metric_visualization/metric_config_panel/visual_options_popover.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiHorizontalRule } from '@elastic/eui';
+import { ToolbarPopover, TooltipWrapper } from '../../shared_components';
+import { TitlePositionOptions } from './title_position_option';
+import { FramePublicAPI } from '../../types';
+import { MetricState } from '../../../common/expressions';
+import { TitleAlignOptions } from './title_align_option';
+import { TitleSizeOptions } from './title_aize_option';
+
+export interface VisualOptionsPopoverProps {
+  state: MetricState;
+  setState: (newState: MetricState) => void;
+  datasourceLayers: FramePublicAPI['datasourceLayers'];
+}
+
+export const VisualOptionsPopover: React.FC<VisualOptionsPopoverProps> = ({ state, setState }) => {
+  return (
+    <TooltipWrapper tooltipContent={'what'} condition={true}>
+      <ToolbarPopover
+        title={i18n.translate('xpack.lens.shared.curveLabel', {
+          defaultMessage: 'Visual options',
+        })}
+        type="visualOptions"
+        groupPosition="left"
```

Comment:
To avoid showing sharp corners on the right side of this toolbar button, please remove this `groupPosition` prop (as it's the only button in this group).

![image](https://user-images.githubusercontent.com/3884767/151877078-b2a58ed8-547a-497f-8889-78404654224e.png)